aibolit/metrics/cognitiveC/cognitive_c.py

```diff
@@ -2,36 +2,65 @@
 from aibolit.patterns.var_middle.var_middle import JavalangImproved
 from typing import List
 
+increment_for = [
+    javalang.tree.IfStatement,
+    javalang.tree.SwitchStatement,
+    javalang.tree.ForStatement,
+    javalang.tree.WhileStatement,
+    javalang.tree.DoStatement,
+    javalang.tree.CatchClause,
+    javalang.tree.BreakStatement,
+    javalang.tree.ContinueStatement,
+    javalang.tree.TernaryExpression,
+    # javalang.tree.BinaryOperation,
+]
+
+nested_for = [
+    javalang.tree.IfStatement,
+    javalang.tree.SwitchStatement,
+    javalang.tree.ForStatement,
+    javalang.tree.WhileStatement,
+    javalang.tree.DoStatement,
+    javalang.tree.CatchClause,
+]
+
 
 class CognitiveComplexity:
     '''
     beta version of Cognitive Complexity
     '''
     def __init__(self):
-        pass
+        self.complexity = 0
 
-    def if_statements(self, nodes: List) -> int:
-        count = 0
-        for i in nodes:
-            if type(i.node) in [javalang.tree.IfStatement]:
-                count += 1
-        return count
+    def traverse_childs(self, block: javalang.tree, nested_level: int):
+        for each_child in block.children:
+            self.get_complexity(each_child, nested_level)
 
-    def for_statements(self, nodes: List) -> int:
-        count = 0
-        for i in nodes:
-            if type(i.node) in [javalang.tree.ForStatement]:
-                count += 1
-        return count
+    def get_complexity(self, childs, nested_level: int):
```

Comment:
the first argument must have type `javalang.tree.Node`

Reply:
and user more suitable name, not `childs`

Reply:
done